tests/Utils/Image.send.phpt

```diff
@@ -20,5 +22,5 @@ $main = Image::fromFile('images/logo.gif');
 
 
 Assert::exception(function () use ($main) { // invalid image type
-	$main->send('jpg');
-}, 'Nette\InvalidArgumentException', 'Unsupported image type \'jpg\'.');
+	$main->send(IMG_WBMP);
```

Comment:
The parameter should be integer, string `jpg` throws `TypeError`. However I can add another test for the `TypeError` exception.